Task-1-Data-Extraction-Preprocess/books/যা_দেখেছি_যা_বুঝেছি_যা_করেছি/যা_দেখেছি_যা_বুঝেছি_যা_করেছি_1_clean.txt

```diff
@@ -0,0 +1,2157 @@
+
+
+ 
+
+মোনেম খার ছেলে খসরু আইয়ূব বিরোধীদের খতম করার জন্য গুন্ডা বাহিনী গড়ে
+তুলেছিল! রাজনীতিতে ছেলে-ভাগ্নেদের প্রভাব কখনো কল্যাণকর হয়নি। অচিরেই
+মোনেম খাঁর অন্যায় গণবিরোধী কার্যকলাপের জন্য পূর্ব বাংলার জনগণের ক্ষোভ
+আইয়ূব প্রশাসনের বিরুদ্ধে ধুমাঁয়িত হয়ে উঠে ৷
+
+১৯৬৫ সালের পাক-ভারত যুদ্ধের পর থেকে পাকিস্তানের রাজনৈতিক আবহাওয়া
+আরো উত্তপ্ত হয়ে উঠে । এ সময় কতগুলো অপ্রত্যাশিত বিশেষ গুরুত্বপূর্ণ ঘটনা ঘটে
+রাজনৈতিক প্রেক্ষাপটে : পরবর্তী পর্যায়ে ঘটনাগুলো পাকিস্তানের রাজনীতিকে
+বিশেষভাবে প্রভাবিত করে! যুদ্ধ শেষে আইয়ূব খান ও লাল বাহাদুর শাস্ত্রির মধ্যে
+এঁতিহাসিক তাসখন্দ চুক্তি স্বাক্ষরিত হয়! এই চুক্তিতে স্বাক্ষর দানের ফলে পশ্চিম
+পাকিস্তানের আপামর জনসাধারণ আইয়ূব খানের বিরুদ্ধে বিক্ষুব্ধ হয়ে উঠে ।
+
+তাসখন্দ চুক্তির বিরোধিতা করে তৎকালীন পররাষ্ট্রমন্ত্রী জনাব জুলফিকার আলী ভুট্টো
+জাতিসংঘের সাধারণ অধিবেশনে বক্তব্য রাখেন এবং নাটকীয়ভাবে অধিবেশন কক্ষ
+ত্যাগ করেন৷ একই কারণে তিনি তার মন্ত্রীত্বের পদেও ইস্তফা দেন ! এরপর তিনি
+সরাসরিভাবে সক্রিয় রাজনীতিতে অবতীর্ণ হন৷ তিনি পাকিস্তান পিপলস্‌ পার্টি (পি পি
+পি) নামে একটি রাজনৈতিক দল গঠন করে আইয়ূব বিরোধী আন্দোলনে সোচ্চার
+হন। অতি অল্প সময়ের মধ্যেই জনাব ভুষ্টো ও তার দল পিপিপি অস্বাভাবিক
+জনপ্রিয়তা অর্জন করতে সমর্থ হয়। জনাব ভুষ্টো রাজনৈতিক ব্যক্তিত্ব হিসেবে
+আকর্ষণীয় হয়ে উঠেন তরুণ ও ছাত্র সমাজের কাছে । তার ডাকে বিশেষভাবে সাড়া
+দেয় পশ্চিম পাকিস্তানের ছাত্র সমাজ ৷ নবগঠিত পিপিপি-কে জনপ্রিয় করে তুলতে
+এবং পশ্চিম পাকিস্তানে আইয়ূব বিরোধী আন্দোলনকে গতিশীল করে তুলতে তরুণ
+ছাত্র সমাজই অগ্রণী ভূমিকা পালন করে। ১৯৬৮ সালের শেষাশেষি আইয়ূব বিরোধী
+আন্দোলন পিপিপি-র নেতৃত্বে সমগ্র পশ্চিম পাকিস্তানে গণআন্দোলনের রূপ নেয়।
+
+যুদ্ধের সময় পূর্ব পাকিস্তান পশ্চিম পাকিস্তান থেকে সম্পূর্ণ বিচ্ছিন্ন হয়ে যায়। দেশবক্ষা
+বাহিনীর প্রয়োজনীয় ইউনিট এবং অস্ত্র সম্ভারের অভাবে ভারতীয় আক্রমণের মুখে ৩
+দিন টিকে থাকার মত ব্যবস্থাও সেদিন পূর্ব পাকিস্তানের ছিল না ' পশ্চিম পাকিস্তানের
+উপর নির্ভরতার জন্য মাত্র ১৭ দিনের যুদ্ধে পূর্ব পাকিস্তানের বৈদেশিক বাণিজ্য অচল
+হয়ে যায় । এই অবস্থায় আওয়ামী লীগ প্রধান শেখ মুজিবর রহমান পূর্ব পাকিস্তানকে
+অর্থনৈতিক, দেশরক্ষা এবং বৈদেশিক বাণিজ্যের ক্ষেত্রে স্বয়ংসম্পূর্ণ ইউনিট হিসাবে
+গড়ে তোলার জন্য ৬ দফা দাবি পেশ করলেন! তখন তিনি অবশ্য এ কথাও
+বলেছিলেন যে. ৬ দফার প্রশ্নে আলোচনা করা যেতে পারে।
+
+যা দেখেছি, যা বুঝেছি, যা করেছি ৩
+
+
+ 
+
+ছয় দফা :
+
+(0১) এঁতিহাসিক লাহোর প্রস্তাবের ভিত্তিতে পাকিস্তানকে একটি সত্যিকারের
+ফেডারেশন রূপে গড়িতে হইবে ৷ তাহাতে পালামেন্ট পদ্ধতির সরকার থাকিবে । সকল
+নির্বাচন সার্বজনীন প্রাপ্ত বয়স্কদের সরাসরি ভোটে অনুষ্ঠিত হইবে । আইন সভাসমূহের
+সার্বভৌমত্ব থাকিবে।
+
+সংশোধিত: সরকারের চরিত্র হবে ফেডারেল এবং সংসদীয় গণতান্ত্রিক। জাতীয় এবং
+ফেডারেটিং ইউনিটসমূহতে প্রতক্ষ্য গণভোটের মাধ্যমে সরকার নির্বাচিত হবে।
+জনসংখ্যার অনুপাতে ফেডারেল সংসদে প্রতিনিধিত্ব করা হবে।
+
+(২) কেন্দ্রিয় সরকারের হাতে মাত্র দুইটি বিষয় থাকিবে । প্রতিরক্ষা এবং বৈদেশিক
+নীতি । অবশিষ্ট বিষয়সমূহ স্টেট বা প্রদেশসমূহের হাতে থাকিবে ।
+
+(৩) পূর্ব এবং পশ্চিম পাকিস্তানের দুইটি পৃথক অথচ বিনিময় যোগ্য মুদ্রার প্রচলন
+করিতে হইবে৷ অথবা দুই অঞ্চলের জন্য একই কারেঙ্গি থাকিবে ৷ শাসনতন্ত্রে এমন
```

Comment:
কারেঙ্গি  should be কারেন্সি  In avro it will be 'karensi' (There are same "engi" to "ensi" transformation needed in other places in the text)

Reply:
fixed